application/src/test/java/org/opentripplanner/routing/algorithm/mapping/CarPickupSnapshotTest.java

```diff
@@ -0,0 +1,71 @@
+package org.opentripplanner.routing.algorithm.mapping;
+
+import au.com.origin.snapshots.junit5.SnapshotExtension;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.opentripplanner.model.GenericLocation;
+import org.opentripplanner.model.modes.ExcludeAllTransitFilter;
+import org.opentripplanner.routing.api.request.RouteRequest;
+import org.opentripplanner.routing.api.request.StreetMode;
+import org.opentripplanner.routing.api.request.request.filter.AllowAllTransitFilter;
+
+@ExtendWith(SnapshotExtension.class)
+public class CarPickupSnapshotTest extends SnapshotTestBase {
+
+  static GenericLocation p0 = new GenericLocation(
+    "SE Stark    St. & SE 17th Ave. (P0)",
+    null,
+    45.519320,
+    -122.648567
+  );
+
+  static GenericLocation p1 = new GenericLocation(
+    "SE Morrison St. & SE 17th Ave. (P1)",
+    null,
+    45.51726,
+    -122.64847
+  );
+
+  static GenericLocation p2 = new GenericLocation(
+    "NW Northrup St. & NW 22nd Ave. (P2)",
+    null,
+    45.53122,
+    -122.69659
+  );
+
+  @BeforeAll
+  public static void beforeClass() {
+    loadGraphBeforeClass(false);
+  }
+
+  @Test
+  public void test_trip_planning_with_car_pickup_only() {
```

Comment:
We usually don't use the camel case in tests. I know some people like to use them since they might be slightly more readable. Is there some other reason to use them here?

Reply:
The existing snapshots tests follow the same format, which I built on:
https://github.com/opentripplanner/OpenTripPlanner/blob/5b5d92fe7e81c2fab07f2ff6ae8f7488427be71c/application/src/test/java/org/opentripplanner/routing/algorithm/mapping/TransitSnapshotTest.java#L66-L67

Reply:
We discussed this in the dev meeting today. Apparently we had made some decision a couple of years ago that snake case is not allowed in tests. But since it's used in snapshot tests, we don't think you should change it in the scope of this pr but if you could create a follow up pr where you rename all the snake case snapshot tests, that would be appreciated.